bin/git-bulk

```diff
@@ -70,7 +70,7 @@ function guardedExecution () {
 
 # atomic git command execution with log
 function atomicExecution () {
-  echo 1>&2 "${bldred}->${reset} executing ${inverse}git $gitcommand${reset}" && git "$@"
+  echo 1>&2 "${bldred}->${reset} executing in ${gitrepodir} ${inverse}git $gitcommand${reset}" && git "$@"
```

Comment:
It always shows `-> executing in . git summary` no matter what the git repo is.

Reply:
That's strange 🤔 It works works for me:

```bash
$ mkdir foo && cd foo
$ git clone https://github.com/tj/git-extras
$ git clone https://github.com/git/git
$ git clone https://github.com/hashicorp/terraform
$ git bulk --addworkspace test $PWD
$ git bulk status
Core command "status" accepted.
-> executing in ./terraform git status
On branch main
Your branch is up to date with 'origin/main'.

nothing to commit, working tree clean
-> executing in ./git git status
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean
-> executing in ./git-extras git status
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean
```

Reply:
But it doesn't work for me:

```
git bulk -a log --oneline -n 1
Core command "log" accepted.
-> executing in . git log --oneline -n 1
7653b48 (HEAD -> pr/1011) Log in which subfolder a git command get's executed
-> executing in . git log --oneline -n 1
6da667d (HEAD -> master, origin/master, origin/HEAD) ww
```

Reply:
I will take a look at it in the new year.